.vscode/settings.default.json

```diff
@@ -1,5 +1,5 @@
 {
-  // Please keep this file in sync with settings in home-assistant/.devcontainer/devcontainer.json
+  // Please keep this file (mostly!) in sync with settings in home-assistant/.devcontainer/devcontainer.json
```

Comment:
Mostly? not sure why it was added or what "mostly" even entails.

../Frenck

Reply:
Just copy-pasting this reply here to close it:

Im not sure why the line is there either, but now this value no longer is
in sync, which is why I added the "mostly"